src/Parser/Mage/Fire/Modules/Checklist/Component.js

```diff
@@ -0,0 +1,115 @@
+          {combatant.hasTalent(SPELLS.PHOENIX_FLAMES_TALENT.id) && <Requirement name="Phoenix Flames Usage" thresholds={thresholds.phoenixFlamesHeatingUpUsage} tooltip="Because Phoenix Flames is guaranteed to crit, you should only use it to convert Heating Up into Hot Streak." />}
+          <Requirement name="Fire Blast Usage" thresholds={thresholds.fireBlastHeatingUpUsage} tooltip="Because Fire Blast is guaranteed to crit, you should only use it to convert Heating Up into Hot Streak." />
+        </Rule>
+        <Rule
+          name="Use your talents effectively"
+          description="Regardless of which talents you select, you should ensure that you are utilizing them properly. If you are having trouble effectively using a particular talent, you should consider taking a different talent that you can utilize properly or focus on effectively using the talents that you have selected."
+        >
+          {combatant.hasTalent(SPELLS.PYROCLASM_TALENT.id) && <Requirement name="Pyroclasm Utilization" thresholds={thresholds.pyroclasmUtilization} tooltip="Pyroclasm has a chance to give you a buff that makes your next non instant Pyroblast deal 225% additional damage. You should ensure that you are using these procs (especially during Combustion) somewhat quickly to ensure you dont waste or overwrite any of these procs." />}
+          {combatant.hasTalent(SPELLS.SEARING_TOUCH_TALENT.id) && <Requirement name="Searing Touch Utilization" thresholds={thresholds.searingTouchUtilization} tooltip="Searing Touch causes your Scorch ability to deal 150% additional damage and be guaranteed to crit when the target is under 30% health. Therefore it is important that when the target is under 30% health, you cast Scorch instead of Fireball." />}
+          {combatant.hasTalent(SPELLS.RUNE_OF_POWER_TALENT.id) && <Requirement name="Rune of Power Uptime" thresholds={thresholds.runeOfPowerBuffUptime} tooltip="Using Rune of Power effectively means being able to stay within the range of it for it's entire duration. If you are unable to do so or if you frequently have to move out of the range of the buff, consider taking a different talent instead." />}
+        </Rule>
+        <Rule
+          name="Avoid downtime"
+          description={(
+            <React.Fragment>
+              As a DPS, it is important to spend as much time casting as possible as if you arent casting you arent doint damage. Therefore it is important to minimize your movements, stay within range of the target, and cancelling casts .. if you can avoid it. While some fights will have an amount of time that is unavoidable downtime; the more you can minimize that downtime, the better.
```

Comment:
`arent` -> `are not` or `aren't` x2

Reply:
The last sentence of this one is also awkward, but I'm not going to block the PR over that.

Reply:
fixed.